modules/appeals_api/app/controllers/appeals_api/notice_of_disagreements/v0/notice_of_disagreements_controller.rb

```diff
@@ -34,18 +32,19 @@ class NoticeOfDisagreementsController < AppealsApi::V2::DecisionReviews::NoticeO
     }.freeze
 
     def schema
-      response = AppealsApi::JsonSchemaToSwaggerConverter.remove_comments(
-        AppealsApi::FormSchemas.new(
-          SCHEMA_ERROR_TYPE,
-          schema_version: self.class::SCHEMA_VERSION
-        ).schema(self.class::FORM_NUMBER)
-      )
-
-      render json: response
+      render json: AppealsApi::JsonSchemaToSwaggerConverter.remove_comments(form_schemas.schema(FORM_NUMBER))
     end
 
     private
 
+    def form_schemas
```

Comment:
Need to memoize this - I will update this PR to create a concern that will do this work instead